Sources/PointFree/Account/Invite.swift

```diff
@@ -14,7 +14,7 @@ let showInviteMiddleware =
   requireTeamInvite
     <| currentUserMiddleware
     >-> writeStatus(.ok)
-    >-> respond(showInviteView)
+    >-> respond(showInviteView.contramap(lower))
```

Comment:
realized we dont need to use `Tuple`'s in views cause we can just `contramap(lower)` to get a proper tuple that can be destructured in the view function